AfterpayTests/PriceBreakdownTests.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  PriceBreakdownTests.swift
+//  AfterpayTests
+//
+//  Created by Adam Campbell on 10/8/20.
+//  Copyright © 2020 Afterpay. All rights reserved.
+//
+
+@testable import Afterpay
+import XCTest
+
+class PriceBreakdownTests: XCTestCase {
+
+  override func tearDown() {
+    Afterpay.setConfiguration(nil)
+  }
+
+  func testNoConfiguration() {
+    let breakdown = PriceBreakdown(totalAmount: .zero)
+
+    XCTAssertEqual(breakdown.badgePlacement, .end)
+    XCTAssertEqual(breakdown.string, "or pay with")
+  }
+
+  private let setMinumumMaximumConfiguration = {
```

Comment:
Is there a reason this isn't a `private func` or is it just a style preference?

Reply:
Yeah, it could easily be a function. Style choice, the fact that functions in test cases that start with `test` are treated differently to different functions is a little nuanced. This might be a little more clear that it's not a test function.